packages/editor/src/editor/code/CodePane.tsx

```diff
@@ -44,13 +44,8 @@ export const CodePane = (props: CodePaneProps) => {
   );
 
   useEffect(() => {
-    const mounted = mount({
-      [KeyboardActions.Save]: () => {
-        onChange(temporaryString);
-      },
-    });
-    return mounted.dispose;
-  }, [temporaryString]);
+    onChange(debouncedTemporaryString);
```

Comment:
thanks to this, creator now sees changes made to the code with delay specified in useDebouncedValue hook